src/base58.cpp

```diff
@@ -88,34 +90,42 @@ static const int log256_58Ratio = ceil((std::log(256) / std::log(58)) * baseScal
     return true;
 }
 
-std::vector<unsigned char> Clone(const Span<const unsigned char>& input, size_t leadingZeros) {
-    return std::vector<unsigned char>(input.begin() + leadingZeros, input.end());
+auto ConvertToLongArray(const Span<const unsigned char>& input, size_t leadingZeros, size_t groupSize) -> std::vector<int64_t> {
```

Comment:
converting from `base(256)` to `base(256^7)` speeds up processing since we can do actual division and modulo on the resulting int64